### PR TITLE
Fix Gradle lockfiles.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

**Review by gmackall — APPROVED** [review]

**Comment by zanderso** [comment]
If/when this needs to be re-landed, please include an explanation in the PR description or a link to a GitHub issue. Thanks!

File: dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart [hunk 1]
```diff
@@ -14,14 +14,14 @@ import 'package:intl/intl.dart' as intl;
 import 'stock_strings_en.dart';
 import 'stock_strings_es.dart';
 
-/// Callers can lookup localized strings with an instance of StockStrings returned
-/// by `StockStrings.of(context)`.
+/// Callers can lookup localized strings with an instance of StockStrings
```
Comment (zanderso) [inline dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart:17]: The changes in this file look unrelated to updating the lock files. Should it go in a separate PR?

File: dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart [hunk 2]
```diff
@@ -134,16 +134,16 @@ class _StockStringsDelegate extends LocalizationsDelegate<StockStrings> {
...
 
   // Lookup logic when language+country codes are specified.
   switch (locale.languageCode) {
     case 'en': {
-      switch (locale.countryCode) {
-        case 'US': return StockStringsEnUs();
-      }
-      break;
-    }
+  switch (locale.countryCode) {
```
Comment (zanderso) [inline dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart:142]: The indentation here still doesn't look right.

File: dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart [hunk 3]
```diff
@@ -97,7 +97,7 @@ abstract class StockStrings {
   static const List<Locale> supportedLocales = <Locale>[
     Locale('en'),
     Locale('en', 'US'),
-    Locale('es'),
+    Locale('es')
```
Comment (zanderso) [inline dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart:100]: I believe the style guide asks to keep the trailing comma here, but double check me on that.
